aerosols/tholins.py

```diff
@@ -54,7 +54,7 @@ def fname(self, fname):
         if not self.__fname.exists():
             raise FileNotFoundError(f"Database not found: {self}")
 
-        self.con = sqlite.connect(self.fname)
+        self.con = sqlite.connect(str(self.fname))
```

Comment:
These parenthesis are no longer required since python 3.7
The current min-requirement is 3.8